Frends.FTP.ReadFile/Frends.FTP.ReadFile/Definitions/Connection.cs

```diff
@@ -0,0 +1,200 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.FTP.ReadFile.Definitions;
+
+/// <summary>
+/// FTP transport types.
+/// </summary>
+public enum FtpTransportType
+{
+    /// <summary>
+    /// Binary transport.
+    /// </summary>
+    Binary,
+
+    /// <summary>
+    /// ASCII transport.
+    /// </summary>
+    Ascii,
+}
+
+/// <summary>
+/// FTP connection modes.
+/// </summary>
+public enum FtpMode
+{
+    /// <summary>
+    /// Passive mode
+    /// </summary>
+    Passive,
+    /// <summary>
+    /// Active mode.
+    /// </summary>
+    Active,
+}
+
+/// <summary>
+/// FTPS encryption modes.
+/// </summary>
+public enum FtpsSslMode
+{
+    /// <summary>
+    /// No encryption (plain text).
+    /// </summary>
+    None,
+
+    /// <summary>
+    /// Use explicit encryption.
+    /// </summary>
+    Explicit,
+
+    /// <summary>
+    /// Use implicit encryption.
+    /// </summary>
+    Implicit,
+
+    /// <summary>
+    /// Tries to use FTPS encryption and falls back to plain text FTP.
+    /// </summary>
+    Auto,
+}

```

Comment:
There appears to be an "Enums" folder. Would it be possible to move these enums to that folder, or alternatively, move all into a single file?